apps/studio/components/layouts/TableEditorLayout/EntityListItem.tsx

```diff
@@ -140,6 +140,67 @@ const EntityListItem: ItemRenderer<Entity, EntityListItemProps> = ({
     }
   }
 
+  const exportTableAsSQL = async () => {
```

Comment:
is this to export the table or the table rows? might be more accurate to label the dropdown menu as export table data perhaps? i initially thought it'll include the "CREATE table" syntax

Reply:
also can we try to reuse the function that you wrote under "Header.tsx"? at first glance they look like they can share the same function (and we can write one test to cover these two cases)